docs/static/setup.sh

```diff
@@ -11,7 +11,7 @@ title="ZMK Config Setup:"
 # TODO: Check for user.name and user.email git configs being set
 
 prompt="Pick an MCU board:"
-options=("nice!nano" "QMK Proton-C")
+options=("nice!nano" "QMK Proton-C" "BlueMicro52840 (v1)")
```

Comment:
Still missing the extra section in the `select` block right below this.

Reply:
Added.